docs/src/main/sphinx/admin/properties-task.rst

```diff
@@ -111,6 +111,32 @@ resource utilization, but uses additional memory.
 The minimum number of drivers guaranteed to run concurrently for a single task given
 the task has remaining splits to process.
 
+``task.scale-writers``
```

Comment:
Having this once in "Writer scaling properties" should be sufficient, no need to repeat

Reply:
I think it's better to have it here also. To keep the information complete and better discoverability

Reply:
I don't think we repeat descriptions of properties in the docs. It makes future maintenance of docs more difficult. You can try adding reference links to from here to the relevant properties in writer scaling page, but repeating the description in two places is probably not a good idea.
cc: @mosabua

Reply:
Agreed .. we should NOT repeat .. instead just link to the relevant section .

Reply:
@mosabua What's the best way to add a link to the config property. Can you point me to an example?

Reply:
I've added a link